lua/lambdaplayers/lambda/weapons/hl2_crossbow.lua

```diff
@@ -14,52 +15,49 @@ table.Merge( _LAMBDAPLAYERSWEAPONS, {
 
         clip = 1,
 
-        reloadtime = 1.8,
-        reloadanim = ACT_HL2MP_GESTURE_RELOAD_AR2,
+        reloadtime = 1.83,
         reloadanimationspeed = 1,
         reloadsounds = { 
-            { 1, "weapons/crossbow/bolt_load"..random(2)..".wav" } 
+            { 0, "Weapon_Crossbow.Reload" },
```

Comment:
That's the HL1 reload sound, the crossbow only has the electrify sound for the player.